### DISCUSSION TITLE
Obsidian Excalidraw Mobile app navigation

### DISCUSSION THREAD

**Discussion by HongMind** [zsviczian/obsidian-excalidraw-plugin#184] [Q&A]
In the desktop version:
Ctrl + Click =  Open the link in current pane
Ctrl + Shift + Click = Open the link in a new pane.
Ctrl + K = Search for links in the entire Vault

How do we perfrom above commands in the mobile version (ios or android)?

**Comment by zsviczian** [comment]
To follow a link on the mobile you can use the link button in the top right. Just select the text element or image you want to open, then click the link button.
![image](https://user-images.githubusercontent.com/14358394/140557159-5d98f327-4534-4c2f-844d-65a5aef20bf6.png)

For other functions, you can use the command palette by pulling down from the top of the screen on your mobile.
